src/components/Header.tsx

```
@@ -8,30 +8,47 @@ import {
     if (usuarioStr) {
       try {
         const usuario = JSON.parse(usuarioStr)
-        if (usuario?.name) {
-          setNomeUsuario(usuario.name)
+        if (usuario?.nome) {
+          setNomeUsuario(usuario.nome)
         }
       } catch (error) {
         console.error('Erro ao parsear usuário:', error)
-        localStorage.removeItem("user")
+        sessionStorage.removeItem("user")
```

ESLint

[error] Replace `"user"` with `'user'`

(prettier/prettier)